packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -261,6 +261,33 @@ export default class ListControl extends React.Component {
     } else {
       this.props.validate();
     }
+    this.props.onValidateObject(this.props.forID, this.validateSize());
```

Comment:
I noticed that when `required: false` is set and there are no values the validation still fails. I think that in that case we should not validate `min` and `max`, WDYT?

Reply:
see new commit